copper-server-impl/src/main/java/com/pogeyan/cmis/impl/services/CmisObjectService.java

```diff
@@ -2375,22 +2553,67 @@ public static void updateProperties(String repositoryId, Holder<String> objectId
 				if (!(customValues.getKey().equals("cmis:secondaryObjectTypeIds"))) {
 					PropertyData<?> valueName = customValues.getValue();
 					if (valueName.getFirstValue().getClass().getSimpleName().equalsIgnoreCase("GregorianCalendar")) {
-						GregorianCalendar value = convertInstanceOfObject(valueName.getFirstValue(),
-								GregorianCalendar.class);
-						Long time = value.getTimeInMillis();
-						updatecontentProps.put("properties." + valueName.getId(), time.longValue());
+
+						if (!valueName.getFirstValue().toString().equals("")) {
```

Comment:
valueName.getValues().size() > 0 is good enough to check instead of checking with strings.